### PR TITLE
New LinkButton component

### PR DESCRIPTION
**Pull request by salamaashoush**

### CODE REVIEW

**Review by isaozler — APPROVED** [review]

File: packages/libs/react-ui/src/index.ts [hunk 1]
```diff
@@ -122,3 +122,6 @@ export {
   maskValue,
   useDialog,
 } from './components';
+
+// re-exports from 'react-aria'
+export { RouterProvider } from 'react-aria';
```
Comment (eileenmguo) [inline packages/libs/react-ui/src/index.ts:127]: Can you add some documentation or examples of how this component would work with next/link?

Comment (salamaashoush) [reply packages/libs/react-ui/src/index.ts:127]: it is already documented here https://react-spectrum.adobe.com/react-aria/routing.html, I will add the link in storybook

**Comment by eileenmguo** [comment]
I see that my previous comment didn't get posted. Are you planning to remove the old Button component in a separate PR?

**Comment by salamaashoush** [comment]
Yes i am on it already it will be separate MR

**Review by eileenmguo — APPROVED** [review]
I assume there are instances of this being used in the docs. Can we update them as part of this PR?

**Comment by salamaashoush** [comment]
> I assume there are instances of this being used in the docs. Can we update them as part of this PR?

if means the LinkButton/Button I will a separate PR for them, there is no need to block this MR for the apps since it will require more review from owners and also needs a bit of visual test

**Pull request merged**